src/platform_impl/ios/view.rs

```diff
@@ -27,24 +27,49 @@ macro_rules! add_property {
         $name:ident: $t:ty,
         $setter_name:ident: |$object:ident| $after_set:expr,
         $getter_name:ident,
+    ) => {
+        add_property!(
+            $decl,
+            $name: $t,
+            $setter_name: true; |$object| $after_set,
+            $getter_name,
+        )
+    };
+    (
+        $decl:ident,
+        $name:ident: $t:ty,
+        $setter_name:ident: $capability:expr; |$object:ident| $after_set:expr,
+        $getter_name:ident,
     ) => {
         {
             const VAR_NAME: &'static str = concat!("_", stringify!($name));
             $decl.add_ivar::<$t>(VAR_NAME);
-            #[allow(non_snake_case)]
-            extern "C" fn $setter_name($object: &mut Object, _: Sel, value: $t) {
-                unsafe {
-                    $object.set_ivar::<$t>(VAR_NAME, value);
+            let setter = if $capability {
+                #[allow(non_snake_case)]
+                extern "C" fn $setter_name($object: &mut Object, _: Sel, value: $t) {
+                    unsafe {
+                        $object.set_ivar::<$t>(VAR_NAME, value);
+                    }
+                    $after_set
                 }
-                $after_set
-            }
+                $setter_name
+            } else {
+                #[allow(non_snake_case)]
+                extern "C" fn $setter_name($object: &mut Object, _: Sel, value: $t) {
+                    unsafe {
+                        $object.set_ivar::<$t>(VAR_NAME, value);
+                    }
+                    log::warn!(concat!("`", stringify!($getter_name), "` is has no effect on this version of iOS"))
```

Comment:
```suggestion
                    log::warn!(concat!("`", stringify!($getter_name), "` has no effect on this version of iOS"))
```
It would also be good to add the same info that I mentioned in my comment on `maximumFramesPerSecond` - unless plumbing that info through would be super painful?